### ISSUE TITLE
WARNING: Plugin ServerlessWebpack uses deprecated hook before:deploy:createDeploymentArtifact

### ISSUE THREAD

**Issue by alfonsoperez** [serverless-heaven/serverless-webpack#126]
Getting these warnings while deploying my functions:

```
Serverless: WARNING: Plugin ServerlessWebpack uses deprecated hook before:deploy:createDeploymentArtifacts
Serverless: WARNING: Plugin ServerlessWebpack uses deprecated hook after:deploy:createDeploymentArtifacts
```

Serverless version: 1.13.1
Serverless-webpack version: ^1.0.0-rc.4

**Comment by HyperBrain** [comment]
The plugin should hook the new `aws:package:*` events instead.

For a detailed overview see the [lifecycle cheat sheet](https://gist.github.com/HyperBrain/50d38027a8f57778d5b0f135d80ea406)

**Comment by HyperBrain** [comment]
This is a breaking change according to semver as it will require a Serverless version of 1.12+ and will leave users of previous Serverless versions behind.

So I'll remove the 3.1.0 milestone and add it to a breaking 4.0.0 milestone.

**Comment by akolybelnikov** [comment]
@HyperBrain how can I make the plugin hook the new aws:package:* events?   
Serverless Version:     1.23.0
 Serverless-webpack version : ^3.1.2

**Comment by HyperBrain** [comment]
@akolybelnikov This is fixed in #244 . I will prepare a release in the next days, so the warnings are gone.
